content/en/docs/Reference/veneers.md

```diff
@@ -0,0 +1,397 @@
+
+### Schema Goals
+The `semver veneer` must have:
+- terse grammar to minimize creation/maintenance effort
+- deterministic output
+- simple channel promotion for maturing bundles
+- demonstration of a common type of channel maturity model
+- minor-version (Y-stream), major-version (X-stream) versioning optionality
+
+The resulting FBC must clearly indicate how generated channels relate to veneer entities.
+
+### Schema Anatomy
+For convenience and simplicity, this veneer currently supports hard-coded channel names `Candidate`, `Fast`, and `Stable`, in order of increasing channel stability. We leverage this relationship to calculate the default channel for the package. 
+
+`GenerateMajorChannels` and `GenerateMinorChannels` dictate whether this veneer will generate X-stream or Y-stream channels (attributes can be set independently). If omitted, only minor (Y-stream) channels will be generated. 
+
+Under each channel are a list of bundle image references which contribute to that channel. 
+
+With the following (hypothetical) example we define a mock bundle which has 11 versions, represented across each of the channel types:
+```yaml
+Schema: olm.semver
+GenerateMajorChannels: true
+GenerateMinorChannels: true
+Candidate:
+  Bundles:
+  - Image: quay.io/foo/olm:testoperator.v0.1.0
+  - Image: quay.io/foo/olm:testoperator.v0.1.1
+  - Image: quay.io/foo/olm:testoperator.v0.1.2
+  - Image: quay.io/foo/olm:testoperator.v0.1.3
+  - Image: quay.io/foo/olm:testoperator.v0.2.0
+  - Image: quay.io/foo/olm:testoperator.v0.2.1
+  - Image: quay.io/foo/olm:testoperator.v0.2.2
+  - Image: quay.io/foo/olm:testoperator.v0.3.0
+  - Image: quay.io/foo/olm:testoperator.v1.0.0
+  - Image: quay.io/foo/olm:testoperator.v1.0.1
+  - Image: quay.io/foo/olm:testoperator.v1.1.0
+Fast:
+  Bundles:
+  - Image: quay.io/foo/olm:testoperator.v0.2.1
+  - Image: quay.io/foo/olm:testoperator.v0.2.2
+  - Image: quay.io/foo/olm:testoperator.v0.3.0
+  - Image: quay.io/foo/olm:testoperator.v1.0.1
+  - Image: quay.io/foo/olm:testoperator.v1.1.0
+Stable:
+  Bundles:
+  - Image: quay.io/foo/olm:testoperator.v1.0.1
+```
+In this example, `Candidate` has the entire version range of bundles,  `Fast` has a mix of older and more-recent versions, and `Stable` channel only has a single published entry. 
+
+### CLI Tool Usage
+```
+% ./bin/opm alpha render-veneer semver -h
+Generate a file-based catalog from a single 'semver veneer' file
+
+Usage:
+  opm alpha render-veneer semver <filename> [flags]
+
+Flags:
+  -h, --help            help for semver
+  -o, --output string   Output format (json|yaml|mermaid) (default "json")
+
+Global Flags:
+      --skip-tls-verify   skip TLS certificate verification for container image registries while pulling bundles
+      --use-http          use plain HTTP for container image registries while pulling bundles
```

Comment:
Downstream, we would break each command and the output up into separate codeblocks. It helps with copying commands and separating the example output. 

Also, WDYT of putting the flags in a table instead of printing the CLI output? I think it would help with scanability in the doc and make for a reference that is easier to read.